client/benches/tps/config.json

```diff
@@ -1,7 +1,7 @@
 {
     "peers": 4,
-    "interval_us_per_tx": 80000,
-    "max_txs_per_block": 8192,
-    "blocks": 5,
+    "interval_us_per_tx": 10000,
+    "max_txs_per_block": 512,
```

Comment:
Actually this means that we reduce the worst-case latency, which effectively increases the short-run average TPS. It might seem that because it takes non-zero time to vote on a new block, that we'd reduce the long-run average TPS, but you're shuffling less data. The overhead is asymptotically irrelevant.

Reply:
But in practice it is relevant. We wouldn't set `max_txs_per_block` to `1` because that will reduce TPS. The ping time around the network cause rounds in the real world to take time. This means that for maximum throughput you want the largest possible block size. It should be obvious why you need to constrain it the other way though, infinity sized blocks take infinite time to process at each node and so no consensus is ever performed.

The general guide is that the `max_txs_per_block` should be as large as possible while still being able to be processed and pass consensus. My testing suggested 512 was a safe number.